.github/workflows/dart.yml

```diff
@@ -4,12 +4,9 @@ on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     container:
-      image:  google/dart:latest
-
+      image: google/dart:2.12-dev
```

Comment:
Can you add a TODO to revert this to `latest` once `latest` supports NNBD?

Reply:
Done